decryptor/mysql/response_proxy.go

```diff
@@ -174,6 +175,20 @@ func (handler *MysqlHandler) ClientToDbProxy(errCh chan<- error) {
 			firstPacket = false
 			handler.clientProtocol41 = packet.ClientSupportProtocol41()
 			if packet.IsSSLRequest() {
+				if handler.tlsConfig == nil {
+					log.Errorln("To support TLS connections you must pass TLS key and certificate for AcraServer that will be used" +
```

Comment:
Looks like the space was missed at the end of this line (and below in the same block).